README.md

```diff
@@ -347,8 +347,8 @@ class NullFixture extends React.Component {
 
 const nullWrapper = mount(<NullFixture />) // mount/render/shallow when applicable
 
-expect(wrapper).to.be.blank()
-expect(wrapper).to.be.empty // an alias
+expect(nullWrapper).to.be.blank()
+expect(nullWrapper).to.be.empty // an alias
```

Comment:
Can you amend your commit to do this for the `nullWrapper` under the following section too please.